sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/InsecureHashAlgorithm.NetCore.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System.Security.Cryptography;
```

Comment:
This duplicates much of the code tested by InsecureHashAlgorithm on .Net Framework and I think it does not bring much value since the test is more library oriented. 

If you really want to add tests for .Net core I think we should move the code which can be tested by all languages in a file and then have dedicated files per framework just with differences.